hal/tests/TESTS/pin_names/pinvalidate.py

```diff
@@ -14,29 +14,62 @@
 limitations under the License.
 """
 
-import os
-import re
 import argparse
-import pathlib
 import json
+import os
+import pathlib
```

Comment:
`import os` not necessary as you are you using `pathlib` module. The `pathlib` module already contains all of the capabilities of `os` that you are using.